db_json_util.py

```diff
@@ -74,6 +76,63 @@ def main() -> None:
                     else:
                         print(response.text)
 
+# export data using the API
+        if args.export_data:
+            # check if path exists
+            if args.json_chains:
+                response = requests.get(args.url + '/all/chains')
+                if not os.path.exists(path_chains):
+                    os.makedirs(os.path.dirname(path_chains), exist_ok=True)
+                    if response.status_code == 200:
+                        data_chains = response.json()
+                        with open(path_chains, 'w', encoding='utf-8') as f:
+                            json.dump(data_chains, f, indent=4)
+                        print(f'exported chains to {path_chains}')
+                    else:
+                        print(response.text)
+                elif os.path.exists(path_chains):
+                    user_input = input("File already exists for json chain, overwrite? (y/n): ")
+                    if user_input in ['y', 'Y', 'yes', 'Yes', 'YES']:
+                        # make request to get all chains
+                        if response.status_code == 200:
+                            data_chains = response.json()
+                            with open(path_chains, 'w', encoding='utf-8') as f:
+                                json.dump(data_chains, f, indent=4)
+                            print(f'exported chains to {path_chains}')
+                        else:
+                            print(response.text)
+                    else:
+                        print('exiting, no data exported')
```

Comment:
There is a lot of code duplication between this and the previous `if`, could you try to reduce it?